README.md

```diff
@@ -90,7 +92,16 @@ Listing of useful (mostly) public learning resources for machine learning applic
 - [Deep Learning and Reinforcement Learning Summer School, Toronto, Canada](https://dlrlsummerschool.ca/) (July 25 - August 3, 2018)
 - [PAISS: Artificial Intelligence Summer School, Grenoble, France](https://project.inria.fr/paiss/) (July 2-6, 2018)
 
+##### Upcoming:
+
+- [Machine Learning Summer School 2018, Madrid, Spain](http://mlss.ii.uam.es/mlss2018/index.html) (August 27 - September 7, 2018)
+- [Machine Learning Summer School 2019, Stellenbosch, South Africa](http://mlssafrica.com/) (January 7-18, 2019)
```

Comment:
Kinda reiterating the above comment, do you know information on this summer school program on why it might be of interest for HEP people? Or do you know an organizer?

Reply:
So the reasoning was that there are very few well-known schools (to my knowledge) that focus specifically on MLHEP. MLSS Madrid (given that I'm attending), and certain past iterations of @matthewfeickert MLSS events (based on their video lectures) have focused mainly on Stats, ML, DL, CV, and invited talks. These lectures are non-specific and can be applied to a broad range of fields, especially particle physics (given that we have a LOT of variation in the problems we work on). This is why I felt it would be useful for the MLHEP community to be aware of such schools. 

In the event that we have more schools popping up *explicitly* focusing on ML for HEP, we can edit these out. But until then, I do believe these are events worth attending for ML folks working in HEP, especially to understand broader applications before narrowing down models that could work for HEP.